content/kotlin/concepts/enums/enums.md

```diff
@@ -0,0 +1,75 @@
+---
+Title: 'Enums'
+Description: 'Enums are a special data type that enable a variable to be a set of predefined constants.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Enums'
+  - 'Data Types'
+  - 'Data Structures'
+CatalogContent:
+  - 'learn-kotlin'
+  - 'paths/computer-science'
+---
+
+**Enums** are a special data type that enable a variable to be a set of predefined constants.
+
+Kotlin enums are called enum classes and they aren't just mere collections of constants, they also have properties, methods, etc. Each enum acts as a separate instance of the enum class, and the declaration begins with the keywords `enum class` followed by the class name.
```

Comment:
```suggestion
Kotlin enums are called enum classes and they aren't simply collections of constants, as in other languages, they also have properties, methods, etc. Each enum acts as a separate instance of the enum class, and the declaration begins with the keywords `enum class` followed by the class name.
```

Reply:
Since enums in kotlin have additional many additional features, which are being highlighted here in the description, we should also explain what they are and how they work in more detail in this entry. 

Additionally, it appears that a lot of the content here is very similar to the this site: https://blog.logrocket.com/kotlin-enum-classes-complete-guide/

While it can beneficial to review other documentation for reference or inspiration, all content within submitted entries should be original. Plagiarism is not permitted within docs. I would suggest expanding this entry a bit to include some of the elements described above and creating original examples that supports the content.

Reply:
noted this

thanks